internal/http/http.go

```diff
@@ -26,13 +26,19 @@ func New(autoupdate *autoupdate.Autoupdate, auther Auther) *Handler {
 		auther:     auther,
 	}
 	h.mux.Handle("/system/autoupdate", http.HandlerFunc(h.handleAutoupdate))
+	h.mux.Handle("/system/health", http.HandlerFunc(h.handleHealth))
 	return h
 }
 
 func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.mux.ServeHTTP(w, r)
 }
 
+func (h *Handler) handleHealth(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+    fmt.Fprint(w, "{\"healthy\": true}\n")
```

Comment:
Please run [gofmt](https://blog.golang.org/gofmt) before you create a commit. I use vscode with the [ms-vscode.go](https://code.visualstudio.com/docs/languages/go) plugin, with does this automatically. 

I added a [new github action](https://github.com/OpenSlides/openslides3-autoupdate-service/pull/6) to ensure gofmt was called.

A part from this, the code is ok. I would probably use fmt.Fprintln() and use ` instead of " to get rid of the escape characters: 
```
fmt.Fprintln(w, `{"healthy": true}`)
```